app/code/Magento/MediaGalleryUi/Test/Mftf/Test/AdminEnhancedMediaGallerySearchByMultipleKeywordsTest.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminEnhancedMediaGallerySearchByMultipleKeywordsTest">
+        <annotations>
+            <features value="MediaGallery"/>
+            <useCaseId value="https://github.com/magento/adobe-stock-integration/issues/1392"/>
+            <title value="User filter images by their keywords in Standalone Media Gallery"/>
+            <stories value="User filter images by their keywords in Standalone Media Gallery"/>
+            <testCaseId value="https://studio.cucumber.io/projects/131313/test-plan/folders/1337102/scenarios/4503803"/>
+            <description value="User checks search by multiple keywords returns related images"/>
+            <severity value="CRITICAL"/>
+            <group value="media_gallery_ui"/>
+        </annotations>
+        <before>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+            <actionGroup ref="AdminOpenStandaloneMediaGalleryActionGroup" stepKey="openMediaGallery"/>
+        </before>
+        <after>
+            <actionGroup ref="AdminAdobeStockMediaGalleryClearFiltersActionGroup" stepKey="clearFilters"/>
```

Comment:
Adobe Stock action groups cannot be used in magento2. magento2 should not depend on ASI extension
```suggestion
            <actionGroup ref="ClearFiltersAdminDataGridActionGroup" stepKey="clearFilters"/>
```

Reply:
Hi @sivaschenko , I had to remove the clear filter action group instead since it caused an error. The target selector is more than one for `ClearFiltersAdminDataGridActionGroup` unlike `AdminAdobeStockMediaGalleryClearFiltersActionGroup` where it specified the media gallery container grid class. I tested the test file and `ResetAdminDataGridToDefaultViewActionGroup` is enough to reset the grid and clear any filters. Is that alright?